features/src/main/scala/com/salesforce/op/utils/spark/RichVector.scala

```diff
@@ -64,6 +66,16 @@ object RichVector {
       toSpark(res)
     }
 
+    /**
+     * Combine multiple vectors into one
+     *
+     * @param that  another vector
+     * @param other other vectors
+     * @return result vector
+     */
+    def combine(that: Vector, other: Vector*): Vector =
+      com.salesforce.op.utils.spark.RichVector.combine(v +: that +: other)
```

Comment:
do you need the full import here?

Reply:
this is to distinguish the object from the implicit class.